.travis.yml

```diff
@@ -1,8 +1,15 @@
+sudo: false
```

Comment:
it is the default for new projects, but I checked & we were getting the old infrastructure because we're old skool